template-compose/app/src/main/java/co/nimblehq/template/compose/ui/theme/Color.kt

```diff
@@ -2,7 +2,5 @@ package co.nimblehq.template.compose.ui.theme
 
 import androidx.compose.ui.graphics.Color
 
-object Color {
-    val GreenCitrus = Color(0xFF99CC00)
-    val GreenChristi = Color(0xFF669900)
-}
+// Base colors here
+internal val GreenCitrus = Color(0xFF99CC00)
```

Comment:
I feel a bit confused with having both `Color.kt` & `AppColors.kt`

Would it be more clear if we move this to the top of `AppColors.kt`? Then we can remove `Color.kt` 💭

Reply:
@Tuubz 
- `Color` contains the base color without specifying any UI components, such as Black, White, GreenCitrus, etc.
- `AppColors` makes use of colors in the `Color` class above to define custom colors for UI components, such as `AppBarBackground = White` 🤓 

With small to medium projects, it's okay to combine them into 1 class file 👍 We will separate them when needed 👍

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/376/commits/bfe4c1f547c6b64ea983839027bdc70c9c38a6ad